smartcosmos-framework/src/main/java/net/smartcosmos/annotation/SmartCosmosBootstrapConfiguration.java

```diff
@@ -80,16 +79,13 @@ public SendsSmartCosmosEventAdvice sendsSmartCosmosEventAdvice(SmartCosmosEventT
         @Bean
         @Autowired
         @Profile("!test")
-        SmartCosmosEventTemplate smartCosmosEventTemplate(
-            RibbonClientHttpRequestFactory ribbonClientHttpRequestFactory,
-            Executor smartCosmosEventTaskExecutor) {
+        SmartCosmosEventTemplate smartCosmosEventTemplate(Executor smartCosmosEventTaskExecutor) {
```

Comment:
I would expect a new artifact version in the `pom.xml`, because changes like this could break the API.